source/get-started/connect-to-mongodb.txt

```diff
@@ -0,0 +1,77 @@
+.. _railsmdb-get-started-connect-to-mongodb:
+
+==================
+Connect to MongoDB
+==================
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create your Application
+
+      In your shell, run the following command to create a new application by
+      using {+cli-short+}:
+
+      .. code-block:: bash
+
+         railsmdb new quickstart
+
+      This command creates a new folder in your current directory called
+      ``quickstart`` and populates it with the scaffolding for a new application
+      that uses Mongoid with MongoDB.
+
+   .. step:: Assign the Connection String
+
+      In your shell, run the following command to navigate to the
+      ``config/mongoid.yml`` file that was automatically created in the preceding step:
+
+      .. code-block:: bash
+
+         cd config/mongoid.yml
+
+      Open the file in your editor of choice. Then, paste the 
+      connection string that you copied from the :ref:`railsmdb-get-started-connection-string`
+      step of this guide into the ``development.clients.default`` property, as
+      shown in the following example:
+
+      .. code-block:: yaml
+         :emphasize-lines: 4
+
+         development:
+           clients:
+             default:
+               uri: <your connection string URI>
+
+      .. note::
+
+         To use this method to connect your application to your MongoDB Atlas
+         deployment, you must remove the automatically generated
+         ``development.clients.database`` and ``development.clients.hosts``
+         properties.
+
+   .. step:: Generate MVC Scaffolding
+
+      You can use {+cli-short+} to automatically generate a model, view, and
+      controller for a specified object.
```

Comment:
Do we expect the audience to know what model/view/controller means, or should we define them briefly here?

Reply:
Since this tool is meant for people that are already Rails users and familiar with how Rails works, I think we can assume they do. We can also consider adding a brief page somewhere about MVC if needed